packages/confirm-dialog/src/vaadin-confirm-dialog-overlay.js

```diff
@@ -24,6 +24,10 @@ const confirmDialogOverlay = css`
     height: var(--_vaadin-confirm-dialog-content-height);
   }
 
+  ::slotted([slot='header']) {
+    pointer-events: auto;
```

Comment:
The original CSS also applied `display: contents`:

https://github.com/vaadin/web-components/blob/7072364960f0f5be719b41ae6bbd8f3750bd8fe8/packages/dialog/src/vaadin-dialog-styles.js#L24-L29

Perhaps we need it for the confirm dialog as well?

Reply:
IIRC setting `display: contents` was needed to make sure dragging works correctly, but I'm not sure.
Anyways, I would rather avoid making this change in a patch release, as it might break something.